test/powershell/Modules/Microsoft.PowerShell.Utility/Out-File.Tests.ps1

```diff
@@ -127,4 +127,11 @@ Describe "Out-File" -Tags "CI" {
 	# reset to not read only so it can be deleted
 	Set-ItemProperty -Path $testfile -Name IsReadOnly -Value $false
     }
+    It "Creates a file using Out-File with the alias 'Path' for the -filepath parameter" {
+    {Out-File -path .\test.txt | Should Not Throw}
```

Comment:
The 'describe' block already define '$testfile' as test file path and define 'AfterEach' block to remove a test file after each test. So your test would be:
```powershell
    It "Should be able to use 'Path' alias for 'Filepath' parameter" {
	{ Out-File -Path $testfile -InputObject $expectedContent } | Should Not Throw
    }
```